tests/unit/config/test_configinit.py

```diff
@@ -65,6 +67,22 @@ def configdata_init(monkeypatch):
 
 class TestEarlyInit:
 
+    def test_config_py_path(self, args):
+        with tempfile.NamedTemporaryFile(encoding = 'utf-8' ,
```

Comment:
Instead of the `tempfile` module, please use the [`tmpdir` fixture](https://docs.pytest.org/en/latest/tmpdir.html) provided by pytest. The object you get is a [py.path.local](https://py.readthedocs.io/en/latest/path.html) instance, and there's also some [docs about fixtures](https://docs.pytest.org/en/latest/fixture.html).